app/elements/boards-panel/boards-panel.html

```diff
@@ -1,96 +1,68 @@
-<!--
-@license
-Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
--->
-
 <link rel="import" href="../../bower_components/polymer/polymer.html">
-<link rel="import" href="../../bower_components/iron-icons/av-icons.html">
-<link rel="import" href="../../bower_components/paper-fab/paper-fab.html">
-<link rel="import" href="../../bower_components/paper-styles/typography.html">
 <link rel="import" href="../../bower_components/gs-board/dist/components/gs-board.html">
-<script src="../../bower_components/lodash/lodash.min.js"></script>
+<link rel="import" href="../../scripts/parserToBoardAdapter.html">
+
 <dom-module id="boards-panel">
   <template>
-    <div style="text-align: center;">
+
+    <style>
+      .boards-panel {
+        text-align: center;
+      }
+
+      .final-board {
+        margin-top: 20px;
+      }
+    </style>
+
+    <div class="boards-panel">
       <div>
         Tablero inicial
       </div>
       <gs-board size='{ "x": 2, "y": 2 }' options='{ "editable": true }'></gs-board>
 
       <template is="dom-if" if="{{finalState}}">
-        <div style="margin-top: 20px;">
-          Tablero final (resultado)
+        <div class="final-board">
+          Tablero final (resultadooo)
```

Comment:
Era para probar el deploy cuando se mergee jajaja
El jul 29, 2016 9:13 AM, "Franco Leonardo Bulgarelli" <
notifications@github.com> escribió:

> In app/elements/boards-panel/boards-panel.html
> <https://github.com/gobstones/gobstones-web/pull/17#discussion_r72781915>:
>
> >        <div>
> >          Tablero inicial
> >        </div>
> >        <gs-board size='{ "x": 2, "y": 2 }' options='{ "editable": true }'></gs-board>
> >
> >        <template is="dom-if" if="{{finalState}}">
> > -        <div style="margin-top: 20px;">
> > -          Tablero final (resultado)
> > +        <div class="final-board">
> > +          Tablero final (resultadooo)
>
> 😮
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/gobstones/gobstones-web/pull/17/files/dd1c8947aa09e464696acd2c1db5c0a3ebf53147#r72781915>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABjmCCegNGRvnNZfXlGEF2lbJBNUJOmWks5qae52gaJpZM4JX6Wx>
> .
>